src/test/react/SnapshotComponentTesting/PostCommentsDisplay.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import PostCommentsDisplay from '../../../PostCommentsDisplay/components/PostCommentsDisplay.jsx';
+import CommentEditor from '../../../PostCommentsDisplay/components/CommentEditor.jsx';
+import Comment from '../../../PostCommentsDisplay/components/Comment.jsx';
+
+test('PostCommentDisplay renders correctly', () => {
```

Comment:
Why are comment tests here? Seems like this should be part of a separate pull request - this one is nominally about listings not comments.

Reply:
Removed comments tests. Will add them back in when appropriate